frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -125,6 +125,48 @@ if Device:isAndroid() then
     table.insert(common_settings.screen.sub_item_table, require("ui/elements/screen_fullscreen_menu_table"))
 end
 
+if Device:hasKeys() then
+    common_settings.key_navigation = {
+        text = _("Key Navigation"),
```

Comment:
Per style it should be `Key navigation` but could you please change it all to `Navigation`? I would probably also move the Links settings under there but that's not for this PR.

Sorry for only thinking of it just now, but I quickly tried it out and some things pop better when you see them. :-)

Reply:
Alternatively it could be named something like "Taps, swipes and keys" but I can't quite put my finger on the right phrase. This would also include a setting for volume up/down = page up/down (on Android), for example.

I've been meaning to (probably) make that a setting, see #1760.

Reply:
> but could you please change it all to Navigation? I 

Sure.

How do you feel then about moving in Screen->Activate menu and Screen->DIsable double tap ?
Maybe also "Start with" ?